docs/content/components/footer.mdx

```diff
@@ -215,6 +215,18 @@ code_snippets:
             }
           ]
         }
+    - title: Back to Top without JavaScript
+      description: |
+        For sites without JavaScript, you will need to add your own 'back to top' feature to comply with accessibility regulations.
+
+        First create a simple anchor tag at the top of your page. This should be placed at the beginning of where the readable content begins.
+
+        `<a name="top"></a>`
```

Comment:
This is bad guidance.

Not only do we not practice this anywhere on our production builds, specs allow anchors with #top to automatically point to the beginning of the document. 

* https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#sect23
* https://html.spec.whatwg.org/multipage/browsing-the-web.html#scroll-to-the-fragment-identifier
* https://drafts.csswg.org/cssom-view/#scroll-to-the-beginning-of-the-document